Reinstate zoom wheel functionality (BL-5584)

StephenMcConnel opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/bloombooks/bloomdesktop/2261)
<!-- Reviewable:end -->

hatton commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>  with one question.

---

Review status: 0 of 6 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/BloomBrowserUI/bookEdit/js/bloomEditing.ts, line 722 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/2261#-L77642Fgt6X3hkY8EOd:-L77642Fgt6X3hkY8EOe:b-57kmxm) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/31fd6aebb816aa99f8196fddcfb495989c91228d/src/BloomBrowserUI/bookEdit/js/bloomEditing.ts#L722)):*
> ```XML
>     $("body").on("wheel", function (e) {
>         var theEvent = e.originalEvent as WheelEvent;
>         // reviewSlog does this application work?
> ```

Is this comment copied from somewhere? I don't understand it in this context

---

*[src/BloomExe/Workspace/ZoomControl.cs, line 10 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/2261#-L772hHCVzqUhSvTEqSd:-L772hHCVzqUhSvTEqSe:b7n31dz) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/31fd6aebb816aa99f8196fddcfb495989c91228d/src/BloomExe/Workspace/ZoomControl.cs#L10)):*
> ```Smalltalk
> 	{
> 		public const int kMinimumZoom =  30;
> 		public const int kMaximumZoom = 300;	// is 300% a good limit?
> ```

yes 30% - 300% matches firefox

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/2261#-:-L776LLogA3YfswD-3D0:b-hbp1a3)*
<!-- Sent from Reviewable.io -->

StephenMcConnel commented:
Review status: 0 of 6 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/BloomBrowserUI/bookEdit/js/bloomEditing.ts, line 722 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/2261#-L77642Fgt6X3hkY8EOd:-L7ALwl3Kmt41QH-SQDa:bk3oqlp) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/31fd6aebb816aa99f8196fddcfb495989c91228d/src/BloomBrowserUI/bookEdit/js/bloomEditing.ts#L722)):*
<details><summary><i>Previously, hatton (John Hatton) wrote…</i></summary><blockquote>

Is this comment copied from somewhere? I don't understand it in this context
</blockquote></details>

Yes.  it was a blind copy from the old code.  I've removed the comment.

---

*[src/BloomExe/Workspace/ZoomControl.cs, line 10 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/2261#-L772hHCVzqUhSvTEqSd:-L7AM1CaTaCMagp7_Mtl:b-ywkcfh) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/31fd6aebb816aa99f8196fddcfb495989c91228d/src/BloomExe/Workspace/ZoomControl.cs#L10)):*
<details><summary><i>Previously, hatton (John Hatton) wrote…</i></summary><blockquote>

yes 30% - 300% matches firefox
</blockquote></details>

I've changed the comment to record that.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/2261)*
<!-- Sent from Reviewable.io -->
